libs/dal/src/repositories/layout/layout.schema.ts

```diff
@@ -0,0 +1,54 @@
+import * as mongoose from 'mongoose';
+import { Schema } from 'mongoose';
+
+import { LayoutEntity } from './layout.entity';
+
```

Comment:
There is a mongoose plugin we've been using in some other schemas for the soft delete. You could take a look at feed or integration for example :)

import * as mongooseDelete from 'mongoose-delete';